{
  "endpoint": "/api/authentication/login",
  "method": "POST",
  "request": {
    "headers": {
      "X-Correlation-ID": "7499dbe0-d9f9-4cce-8f93-7578ef02eb0c",
      "X-Test-Context": "{\"featureFile\":\"invalid-credentials.api.feature\",\"scenario\":\"api/features/identity/authentication/edge-cases/invalid-credentials.api.spec.ts\",\"currentStep\":\"Test Execution\",\"testIntent\":\"negative\",\"expectedOutcome\":\"2xx_success\",\"testSessionId\":\"test-1757884771602-nlh2ucrhx\",\"testCase\":\"should handle multiple failed login attempts gracefully\"}"
    },
    "body": {
      "email": "test@example.com",
      "password": "***MASKED***"
    }
  },
  "response": {
    "status": 500,
    "error": {
      "error": "error",
      "message": "Authorization failed",
      "timestamp": "2025-09-14T21:19:31.8498052+00:00"
    }
  },
  "correlationId": "7499dbe0-d9f9-4cce-8f93-7578ef02eb0c",
  "timestamp": {},
  "duration": 192
}